templates/rom_arm9_only/Makefile

```diff
@@ -22,9 +22,9 @@ GAME_ICON	:= icon.bmp
 # --------------------------------
 
 # Root folder of the SD image
-SDROOT		:= sdroot
+SDROOT		?= sdroot
 # Name of the generated image it "DSi-1.sd" for no$gba in DSi mode
-SDIMAGE		:= image.bin
+SDIMAGE		?= image.bin
 
```

Comment:
Undo the changes to the templates, they are supposed to be copied as they are. Leave the changes to the default makefiles, though.